ros2cli/package.xml

```diff
@@ -17,6 +17,7 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_xmllint</test_depend>
+  <test_depend>test_msgs</test_depend>
```

Comment:
Nitpick: insert in alphabetical order.

Reply:
Done in 7750f7c3798d2c427f7ef434a366b631787951f2.